Building ISO fails

### What Happened?

Trying to build the Minikube ISO [as per the docs](https://github.com/kubernetes/minikube/blob/474b52c02fc9f6b3e44dfd1c34fec8f065ab2d44/site/content/en/docs/contrib/building/iso.md?plain=1#L40) yields an error:

```
$ make out/minikube.iso
make: *** No rule to make target 'out/minikube.iso'.  Stop.
```

Looks like the rule was updated in https://github.com/kubernetes/minikube/commit/d92f42a6e459313fe22ca935bc5869cee05146e9 , so seems like the docs just need to be updated.

### Attach the log file

n/a

### Operating System

Redhat/Fedora

### Driver

N/A

--- a/site/content/en/docs/contrib/building/iso.md
+++ b/site/content/en/docs/contrib/building/iso.md
@@ -35,19 +35,26 @@ cd minikube
 
 ### Building
 
+To build for x86
 ```shell
 $ make buildroot-image
-$ make out/minikube.iso
+$ make out/minikube-amd64.iso
+```
+
+To build for ARM
+```shell
+$ make buildroot-image
+$ make out/minikube-arm64.iso
 ```
 
 The build will occur inside a docker container. If you want to do this on
-baremetal, replace `make out/minikube.iso` with `IN_DOCKER=1 make out/minikube.iso`.
-The bootable ISO image will be available in `out/minikube.iso`.
+baremetal, replace `make out/minikube-<arch>.iso` with `IN_DOCKER=1 make out/minikube-<arch>.iso`.
+The bootable ISO image will be available in `out/minikube-<arch>.iso`.
 
 ### Using a local ISO image
 
 ```shell
-$ ./out/minikube start --iso-url=file://$(pwd)/out/minikube.iso
+$ ./out/minikube start --iso-url=file://$(pwd)/out/minikube-<arch>.iso
 ```
 
 ### Modifying buildroot components

